data/Denmark/Folketing/ep-popolo-v1.0.json

```diff
@@ -4660,14 +4660,14 @@
       "contact_details": [
         {
           "type": "email",
-          "value": "min@kum.dk"
+          "value": "min@kum.dk|km@km.dk"
```

Comment:
This doesn't look like a valid email address. I'm not sure where this is coming from, but you'll want to separate these with a semicolon rather than than a pipe, and then the build process will Do The Right Thing.